gobblin-service/src/test/java/org/apache/gobblin/service/modules/orchestration/proc/ResumeDagProcTest.java

```diff
@@ -114,10 +117,8 @@ public void resumeDag() throws IOException, URISyntaxException {
             .filter(a -> a.getMethod().getName().equals("addSpec"))
             .count())
         .sum();
-    long addedDagNodeStates = Mockito.mockingDetails(this.dagManagementStateStore).getInvocations().stream()
-        .filter(a -> a.getMethod().getName().equals("addDagNodeState")).count();
 
     Assert.assertEquals(resumedJobCount, expectedNumOfResumedJobs);
```

Comment:
can't we similarly use `Mockito.verify` and `Mockito.times` here rather than `getInvocations()` and the `.filter` + `.count`?

Reply:
there are multiple objects on which invocations is to be find and summed. not sure if a direct mockito method provides that functionality

Reply:
wouldn't that even be best to check that the spec producer associated w/ the `DagNode` we expect it for is the specific one with its `.addSpec` invoked?